packages/kbn-performance-testing-dataset-extractor/src/es_client.ts

```diff
@@ -43,22 +43,34 @@ interface Transaction {
   duration: { us: number };
 }
 
-export interface Document {
-  labels: Labels;
-  character: string;
-  quote: string;
-  service: { version: string };
+interface Document {
+  '@timestamp': string;
+  labels?: Labels;
   parent?: { id: string };
-  processor: string;
+  service: { name: string; environment: string };
   trace: { id: string };
-  '@timestamp': string;
-  environment: string;
+  transaction: Transaction;
+}
+
+export interface ESQueryDocument extends Omit<Document, 'transaction'> {
```

Comment:
How about we replace `Query` with `Request` (and similarly all other related names introduced in this PR)? Currently we can only use it for queries because the APM agent does not capture request bodies for other types of requests but ultimately we'd like to use this extractor for all requests.

Reply:
I agree, should be fixed now